### PR TITLE
update MLopt

### PR DESCRIPTION
**Pull request by chetanyagoyal**
- add utility function to test script to convert a list of opamp parameter into a list of indices into these range arrays
- fix imports in sky130_nist_tapeout.py

### CODE REVIEW

File: openfasoc/generators/glayout/tapeout/tapeout_and_RL/sky130_nist_tapeout.py [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-
+sys.path.append(path.join(str(Path(__file__).resolve().parents[2])))
 from gdsfactory.read.import_gds import import_gds
 from gdsfactory.components import text_freetype, rectangle
 from glayout.flow.pdk.util.comp_utils import prec_array, movey, align_comp_to_port, prec_ref_center
 from glayout.flow.pdk.util.port_utils import add_ports_perimeter, print_ports
 from gdsfactory.component import Component
 from glayout.flow.pdk.mappedpdk import MappedPDK
-from glayout.flow.blocks.composite.opamp import opamp
+try:
```
Comment (alibillalhammoud) [inline openfasoc/generators/glayout/tapeout/tapeout_and_RL/sky130_nist_tapeout.py:13]: nice, robust :+1:

File: openfasoc/MLoptimization/quickstart.bash
Comment (alibillalhammoud) [inline openfasoc/MLoptimization/quickstart.bash:1]: this does not need to change (just discard changes on this file)

**Review by alibillalhammoud — CHANGES_REQUESTED** [review]
discard changes on the bash script

**Comment by chetanyagoyal** [comment]
@alibillalhammoud thanks for the review, addressed all changes

**Comment by chetanyagoyal** [comment]
@msaligane this is ready to merge

**Review by alibillalhammoud — APPROVED** [review]

**Review by msaligane — APPROVED** [review]

**Pull request merged**